src/main/scala/io/sdkman/changelogs/McsMigration.scala

```diff
@@ -0,0 +1,22 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "065")
+class McsMigration {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_mcs_candidate",
+    author = "mthmulders"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "mcs",
+      name = "MCS",
+      description =
+        "Search the Maven Central Repository from your command line!",
```

Comment:
Since this is what appears on the website and through `sdk list`, perhaps you could flesh this description out a bit. For more examples of what others have done, please have a look at the [SDKs page](https://sdkman.io/sdks) on our website.

Reply:
The feature set really isn't that big at this point, although I have [some ideas](https://github.com/mthmulders/mcs#ideas-for-future-development) for further extension. I assume I can update the description using a new `@ChangeSet def migration002`?

Reply:
@marc0der Have you been in the opportunity to take a second look? Is the description more like your taste? I'm more than willing to extend the description as the feature set grows. It's just not that big yet :-)

Reply:
Sorry for the delay! This is looking good now.